charts/hypha/values.yaml

```diff
@@ -102,18 +100,18 @@ tritoninferenceserver-hypha:
     # s3Endpoint: https://uk1s3.embassy.ebi.ac.uk
     # s3Bucket: s3://model-repository/
     env:
-      - name: TF_FORCE_GPU_ALLOW_GROWTH
-        value: "true"
-      - name: AWS_ACCESS_KEY_ID
-        value: minio
-      - name: AWS_SECRET_ACCESS_KEY
-        value: minio123
-      - name: AWS_REGION
-        value: us-east-1
-      - name: S3_ENDPOINT
-        value: "http://minio:9000"
-      - name: S3_USE_HTTPS
-        value: "0"
+    - name: TF_FORCE_GPU_ALLOW_GROWTH
+      value: "true"
+    - name: AWS_ACCESS_KEY_ID
+      value: minio
+    - name: AWS_SECRET_ACCESS_KEY
+      value: minio123
```

Comment:
I know it doesn't matter but maybe a more sophisticated default value would help? Is there a way to force setting this value?